documentation/checker.md

```diff
@@ -1,6 +1,22 @@
-# Checker workflow (checker.wdl)
-This script runs vcf-to-gds-wf.wdl on a known set of inputs, and validates them against expected outputs with a quick md5sum check. On Terra, both inputs and outputs are pulled from the UCSC-owned topmed_workflow_testing bucket.
+# Checker workflows
+These scripts run pipelines on a known set of inputs, and validates them against expected outputs with a quick md5sum check. On Terra, both inputs and outputs are pulled from the UCSC-owned topmed_workflow_testing bucket. In all cases the expected inputs are provided in JSON files.
 
-This workflow is not simply a copy of the check_gds step. Check_gds would not determine if the unique_variant_ids step ran successfully, instead, it a seperate validation check. Speaking of which, because the inputs are known to not take very long in check_gds, it is enabled *by default.* If you are replacing these files with your own sets of inputs and outputs, and that set is non-downsampled, modern TOPMed data, you may wish to consider skipping this step.
+These workflows are not simply a copy of the check_gds/check_merged_gds steps found in vcf-to-gds and ld-pruning respectively. Instead, they perform a separate validation check using md5 sums.
+
+Due to Cromwell limitations, workflows do not always execute in the order that tasks are written in a WDL file. Efforts were made to reduce in ld-pruning-checker. Similarly, scattered tasks tend to execute their components in a somewhat random order; something like shard-6 --> shard-10 --> shard-20 --> shard-1 is possible even if you have your job concurrency set to 1. For this reason, your pipeline may error out later than you would expect. **Be prepared for the pipeline to run its full course even if you are expecting an error in one of the first md5sum checks!**
```

Comment:
**Efforts were made to reduce in ld-pruning-checker.**

Can we rephrase this? Efforts were made to reduce what?

Reply:
It was meant to explain that the checker has some magic that tries to enforce an order, but tbh looking back it's not only unclear but unnecessary information.